django/forms/boundfield.py

```diff
@@ -277,7 +277,7 @@ def template_name(self):
 
     @property
     def id_for_label(self):
-        return 'id_%s_%s' % (self.data['name'], self.data['index'])
+        return self.data['attrs'].get('id')
```

Comment:
Is `return self.data['attrs'].get('id', 'id_%s_%s' % (self.data['name'], self.data['index']))` better?